src/library-authoring/components/LibraryComponents.tsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable react/prop-types */
```

Comment:
We can remove this now
```suggestion
```

Reply:
Updated: https://github.com/open-craft/frontend-app-course-authoring/pull/41/commits/31012df7989a611fd3658dc8f7ad1159e44e3507